ROADMAP.md

```diff
@@ -1,8 +1,19 @@
 # Roadmap
 
+## An even better tool for development environments
+
+Compose is a great tool for development environments, but it could be even better. For example:
+
+- [Compose could watch your code and automatically kick off builds when something changes.](https://github.com/docker/fig/issues/184)
```

Comment:
As a feature it's a very nice thing to have, but I don't think it needs to be integrated. I think it's better to document it, and provide a recipe to implement it if you're not already familiar with existing tools. There are a LOT of portability issues with file watching and it's not something we need to deal with ourself. There are already tons of existing tools that can handle this for you.